include/direwolf/vulkan/vulkansetup.h

```diff
@@ -12,30 +47,22 @@
 namespace dw {
```

Comment:
I know you previously did nested namespaces `dw::vulkan`, what's the reason not to? I don't know what I like most

Reply:
haha.. me neither.. can't decide ;)
Are there some guidelines?

Reply:
Well we use cpp17 so I'd say nested

Reply:
Well we use cpp17 so I'd say nested

Reply:
works for me